### DISCUSSION TITLE
use leases.json to bulk change static dhcp entries

### DISCUSSION THREAD

**Discussion by bastiitsab** [AdguardTeam/AdGuardHome#6395] [Q&A] [answered]
@schzhn pointed me here (https://github.com/AdguardTeam/AdGuardHome/issues/1700#issuecomment-1798525203) in the right direction of the leases.json. At least I can backup my leases here. But changing here is seems not possible, every change I am doing directly in the json will be overridden after some time from the gui. 

Is there any chance to make changes in the leases.jsons reflect in the gui?

**Comment by ainar-g** [comment, marked as answer]
As with any configuration file changes, you must stop AGH first, then make the change, then restart it.  Otherwise there is no way for AGH to know that changes have been made.

**Reply by bastiitsab** [reply]
you are totally right. thank you :)
